crates/rust-analyzer/src/main_loop.rs

```diff
@@ -115,7 +115,7 @@ pub fn main_loop(
                     Ok(workspace) => loaded_workspaces.push(workspace),
                     Err(e) => {
                         log::error!("loading workspace failed: {:?}", e);
-                        if let Some(ra_project_model::CargoTomlNotFoundError(_)) = e.downcast_ref()
+                        if let Some(ra_project_model::CargoTomlNoneFoundError(_)) = e.downcast_ref()
                         {
                             if !feature_flags.get("notifications.cargo-toml-not-found") {
```

Comment:
NOTE: This feature flag will currently not affect `CargoTomlMultipleFound` and `CargoTomlSearchFileSystemError` which I think can be considerd expected behaviour?